test/service/test_response.py

```diff
@@ -2342,6 +2372,52 @@ def test_matrices_tree(self):
         }
         self.assertEqual(contributor_matrices, one(hit['projects'])['contributorMatrices'])
 
+    def test_matrix_cell_count(self):
+        """
+        Verify analysis file matrix_cell_count values are correctly reported
+        on all endpoints
+        """
+        params = self.params(project_id='90bf705c-d891-5ce2-aa54-094488b445c6')
+        expected_matrix_cell_counts = {
```

Comment:
```suggestion
        expected_counts = {
```

You don't need to be so verbose all the time. It makes the code harder to read, just because there's more of it and the lines tend to get long and need to be wrapped. Elide what's clear from the context.